packages/tools/pie-icons-react/scripts/icon-sizes-map.js

```diff
@@ -0,0 +1,8 @@
+export const iconSizesMap = {
```

Comment:
Can you use the `sizeToClassMap` variable in the config package for this, to save duplicating it here?

Reply:
Good catch. That was a leftover file.